### ISSUE TITLE
add heroku deployment?

### ISSUE THREAD

**Issue by himynameiszim** [atharva-lipare/MS-Teams-Auto-Joiner#7]
i think deploying on heroku or glitch is a perfect idea for those lazy to wake up and turn on the computer to run the script

**Comment by atharva-lipare** [comment]
Yeah but it requires a bit more changes as to make it run on a headless server. I might add it using pyvirtualdisplay, but i can't tell when
